api/napi/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+#![deny(clippy::all)]
```

Comment:
We shouldn't use that in our crates because that would be causing too many hard error when clippy is upgraded.